### DISCUSSION TITLE
How to make an automation simply switches between the fan-speeds

### DISCUSSION THREAD

**Discussion by MacHannes** [michaelarnauts/home-assistant-comfoconnect#27] [General]
Hello Michael!

Thank you so much for your work! I like it very much and use it every day.
I am now trying to control the ventilation system via automations, but can't find the option to set the fan speed to 0,1,2 or 3. Where should I look? Would it be possible for you to provide a sample automation of how you do this at home?
I have the suspicion that you might be doing this with scripts or something like that, which I don't yet understand and can't do. I still prefer to work at the simple level of "automations" at the GUI level, and only rarely with YAML. But once I have access to it, I could go further and adapt it to my story.
I've already described my problem here [click link to HA-community](https://community.home-assistant.io/t/how-to-automate-ventilationspeed-away-low-medium-high-of-the-comfoairq/650273), but maybe I'm the only one who can't figure it out, because no one has responded yet.
I would be very happy to receive a hint, some food for thought.
Many thanks in advance.

Greetings, Hannes

**Comment by michaelarnauts** [comment]
I've replied on your post at the forums.

**Comment by MacHannes** [comment]
Thanks a lot!
